backend/Squirrel.SqlService/Squirrel.SqlService.BLL/Hubs/ConsoleAppHub.cs

```diff
@@ -21,7 +21,7 @@ public override async Task OnConnectedAsync()
         await Clients.Caller.SetClientId(Context.UserIdentifier!);
     }
     
-    public async Task ProcessReceivedDataFromClientSide(Guid queryId, string requestActionName, QueryResultTable queryResultTable)
+    public async Task ProcessReceivedDataFromClientSideAsync(Guid queryId, string requestActionName, QueryResultTable queryResultTable)
```

Comment:
it seems it should be renamed in other places too, because we call this method via `InvokeAsync` by its name 

and also better to move all hardcoded strings to one place as properties to see usings